runtime/j9vm/j7vmi.c

```diff
@@ -556,7 +556,8 @@ JVM_FindLoadedClass(JNIEnv* env, jobject classLoader, jobject className)
 			NULL,
 			J9_JNI_UNWRAP_REFERENCE(className),
 			vmClassLoader,
-			J9_FINDCLASS_FLAG_EXISTING_ONLY);
+			J9_FINDCLASS_FLAG_EXISTING_ONLY,
+			CLASSNAME_VALID);
```

Comment:
This actually adds a call to verifyQualifiedName that wasn't there before. It likely doesn't matter, but could be fixed by adding CLASSNAME_INVALID as an input value to findClassString (i.e. check for this value in there and don't do the verification).

Reply:
Agreed and passed CLASSNAME_INVALID over to findClassString in this case (the check there was updated to avoid verification for invalid bits)